HelpSource/Classes/K2A.schelp

```diff
@@ -7,8 +7,8 @@ categories::  UGens>Conversion
 Description::
 
 To be able to play a control rate UGen into an audio rate UGen,
-sometimes the rate must be converted. K2A converts via linear
-interpolation.
+sometimes the rate must be converted. By default, K2A converts via
```

Comment:
Here's a suggestion for a more verbose description of what K2A does, and what to expect if `ramp=0.0`:

`K2A` will generate audio-rate (code::ar::) samples from a control-rate (code::kr::) input sample. By default, this is done by linearly interpolating from the previous code::kr:: sample to the new sample over the period of the control block (64 samples by default). Therefore, the input value will be reached one block after it is received. This can be disabled by setting code::ramp = 0::, in which case the input value is immediately output and held over the control block (sample-and-hold).